vtex/toolbelt
# Support debugging .Net apps

#### What is the purpose of this pull request?
To give developers a better experience, we are adding support for debugging .Net applications.

#### What problem is this solving?
Develop applications without debugging is hard and time-consuming.  

#### How should this be manually tested?
1. Link a dotnet 2.x app (.Net Core 3.0)
2. Add a breakpoint to your code and call the corresponding endpoint
3. Check if the execution stopped in the breakpoint as expected

#### Screenshots or example usage

#### Types of changes

#### Chores checklist
